.github/workflows/python_unit_tests.yaml

```
@@ -6,6 +6,12 @@ on:
       httpbin_url:
         required: false
         description: Used to set the HTTPBIN_URL environment variable
+    inputs:
+      python-version:
+        description: 'List of Python versions to be used'
```

Quotes have no purpose here

```suggestion
        description: List of Python versions to be used
```